src/generate.js

```diff
@@ -184,48 +191,51 @@ function layersSetup (layersOrder) {
   return layers;
 };
 
-function saveImage (_editionCount)  {
+function saveImage(_editionCount) {
   fs.writeFileSync(`${buildDir}/images/${_editionCount}.png`, canvas.toBuffer('image/png'));
 };
 
 
-function genColor () {
+function genColor() {
   let hue = Math.floor(Math.random() * 360);
   let pastel = `hsl(${hue}, 100%, ${background.brightness})`;
   return pastel;
 };
 
-function drawBackground () {
+function drawBackground() {
   ctx.fillStyle = background.static ? background.default : genColor();
   ctx.fillRect(0, 0, format.width, format.height);
 };
 
-function addMetadata  (_dna, _edition) {
+function addMetadata(_dna, _edition) {
   let dateTime = Date.now();
   let tempMetadata = {
     name: `${namePrefix} #${_edition}`,
     description: description,
-    image: `${baseUri}/${_edition}.png`,
+    content: {
+      aeweb: `${baseUri}/${_edition}.png`,
+    },
+    type_mime: `ìmage/png`,
     dna: sha1(_dna),
     edition: _edition,
     date: dateTime,
```

Comment:
I don't think the date is relevant here, same for edition, dna ...
Or at least it should be parametrable if we want to add them or not